sat/cli/bootsys/parser.py

```diff
@@ -238,6 +238,19 @@ def _add_bootsys_boot_subparser(subparsers):
     _add_bootsys_action_subparser(subparsers, 'boot')
 
 
+def _add_bootsys_reboot_subparser(subparsers: argparse.ArgumentParser):
```

Comment:
I'm not sure if this type hint is actually correct. PyCharm complains with the warning:

> Type 'ArgumentParser' doesn't have expected attribute 'add_parser'

We haven't really started to use type hints in this code base yet, so the easiest way to resolve the warning might be to leave it off.